src/telegram.js

```diff
@@ -845,6 +845,157 @@ class TelegramBot extends EventEmitter {
   }
 
   /**
+   * Use this method to restrict a user in a supergroup.
+   * The bot must be an administrator in the supergroup for this to work
```

Comment:
@kamikazechaser but verbosity is not only in my methods. All code contains "verbosity" descriptions. English not my native to correctly translate all descriptions in one-two descriptions. I think you can merge it and open new issue.